test/system/lifecycle/lifecycle_test.go

```diff
@@ -0,0 +1,311 @@
+			shoot.Spec.Extensions = initialExtensionConfig
+			return nil
+		})
+	})
+
+	f.Serial().Release().CIt("Should perform the common case scenario without any errors", func(ctx context.Context) {
+		err := f.UpdateShoot(ctx, ensureOIDCServiceIsEnabled)
+		Expect(err).ToNot(HaveOccurred())
+
+		oidcDeployment := &appsv1.Deployment{
+			TypeMeta: metav1.TypeMeta{},
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      constants.ApplicationName,
+				Namespace: f.ShootSeedNamespace(),
+			},
+		}
+
+		// Verify that the oidc deployment exists and is deployed with the correct number of replicas
+		err = f.SeedClient.Client().Get(ctx, client.ObjectKeyFromObject(oidcDeployment), oidcDeployment)
+		Expect(err).ToNot(HaveOccurred())
+		one := int32(1)
+		Expect(*oidcDeployment.Spec.Replicas).To(BeNumerically(">=", one))
+		Expect(oidcDeployment.Status.ReadyReplicas).To(BeNumerically(">=", one))
+
+		oidConfig, err := getOIDConfig(ctx, f.SeedClient.RESTClient())
+		if err == nil {
+			jwksURL, err := url.Parse(oidConfig.JWKSURL)
+			Expect(err).ToNot(HaveOccurred())
+
+			jwks, err := getJWKS(ctx, f.SeedClient.RESTClient(), jwksURL.Path)
+			Expect(err).ToNot(HaveOccurred())
+			clientId := "some-custom-client-id"
+			oidcAPIVersion := "authentication.gardener.cloud/v1alpha1"
+			oidcKind := "OpenIDConnect"
+
+			// Deploy oidc resource
+			oidc := &unstructured.Unstructured{}
+			oidc.Object = map[string]interface{}{
+				"apiVersion": oidcAPIVersion,
+				"kind":       oidcKind,
+				"metadata": map[string]interface{}{
+					"name": "custom",
+				},
+				"spec": map[string]interface{}{
+					"issuerURL":      oidConfig.Issuer,
+					"clientID":       clientId,
+					"usernameClaim":  "sub",
+					"usernamePrefix": "custom-prefix:",
+					"jwks": map[string]interface{}{
+						"keys": jwks,
+					},
+				},
+			}
+			err = f.ShootClient.Client().Create(ctx, oidc, &client.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			oidc = &unstructured.Unstructured{}
+			oidc.SetAPIVersion(oidcAPIVersion)
+			oidc.SetKind(oidcKind)
+			oidc.SetName("custom")
+			err = f.ShootClient.Client().Get(ctx, client.ObjectKeyFromObject(oidc), oidc)
+			Expect(err).ToNot(HaveOccurred())
+
+			// Conversion should be safe
+			spec := oidc.Object["spec"].(map[string]interface{})
+			Expect(spec["clientID"]).To(Equal(clientId))
+			Expect(spec["issuerURL"]).To(Equal(oidConfig.Issuer))
+
+			// Get token from seed
+			var ttl int64 = 1800
+			tokenReq, err := f.SeedClient.Kubernetes().CoreV1().ServiceAccounts("default").CreateToken(ctx, "default", &authenticationv1.TokenRequest{
+				Spec: authenticationv1.TokenRequestSpec{
+					Audiences:         []string{clientId},
+					ExpirationSeconds: &ttl,
+				},
+			}, metav1.CreateOptions{})
+			Expect(err).ToNot(HaveOccurred())
+
+			expectedStatus := status{
+				Status:  "Failure",
+				Message: "forbidden: User \"custom-prefix:system:serviceaccount:default:default\" cannot get path \"/\"",
```

Comment:
You can avoid escaping by using back-quote
```suggestion
				Message: `forbidden: User "custom-prefix:system:serviceaccount:default:default" cannot get path "/"`,
```